src/Toolkit/Toolkit.Maui/BasemapGallery/BasemapGallery.Appearance.cs

```diff
@@ -313,6 +308,29 @@ private void HandleTemplateChange(double currentWidth)
         }
     }
 
+    private void UpdateItemsLayout(int span, double verticalSpacing, double horizontalSpacing)
+    {
+        if (ListView is not null)
+        {
+#if __IOS__
+            // This is a workaround for a bug in the current version of the iOS renderer for CollectionView
+            // where CollectionView throws `NullReferneceException` on changing span of GridItemsLayout.
+            ListView.ItemsLayout = new GridItemsLayout(span, ItemsLayoutOrientation.Vertical)
+            {
+                VerticalItemSpacing = verticalSpacing,
+                HorizontalItemSpacing = horizontalSpacing,
+            }; 
+#else

```

Comment:
I have added this to cover [this](https://github.com/dotnet/maui/issues/10144) iOS issue. But it looks like fixed in MAUI version 8.0.10 version. Along with that, There are changes at couple of places where use of `Microsoft.Maui.Platform.MauiDatePicker` support for iOS is removed.
@mstefarov Any idea around it?